src/main.cpp

```diff
@@ -1246,54 +1207,46 @@ void setup() {
                                 .helpText = F("Proportional gain for the steaming mode (I or D are not used)"),
                                 .type = kDouble,
                                 .section = sPIDSection,
-                                .position = 9,
+                                .position = 7,
                                 .show = [] { return true; },
                                 .minValue = PID_KP_STEAM_MIN,
                                 .maxValue = PID_KP_STEAM_MAX,
                                 .ptr = (void*)&steamKp};
 
-    editableVars["TEMP"] = {.displayName = F("Temperature"),
-                            .hasHelpText = false,
-                            .helpText = "",
-                            .type = kDouble,
-                            .section = sPIDSection,
-                            .position = 10,
-                            .show = [] { return false; },
-                            .minValue = 0,
-                            .maxValue = 200,
-                            .ptr = (void*)&temperature};
-
-    editableVars["BREW_SETPOINT"] = {.displayName = F("Set point (°C)"),
-                                     .hasHelpText = true,
-                                     .helpText = F("The temperature that the PID will attempt to reach and hold"),
-                                     .type = kDouble,
-                                     .section = sTempSection,
-                                     .position = 11,
-                                     .show = [] { return true; },
-                                     .minValue = BREW_SETPOINT_MIN,
-                                     .maxValue = BREW_SETPOINT_MAX,
-                                     .ptr = (void*)&brewSetpoint};
+    editableVars["TEMP"] = {
```

Comment:
why are some blocks formatted like before (e.g. for PID_PONM_ON above) and some like this inline? Not sure if this still needs to be auto-formatted

Reply:
i have no clue why this happens, but this is already formated. maybe @simonspa has an idea for it

Reply:
I think if the entire thing fits into the total line length defined, it will not be broken up, and otherwise aligned.

Reply:
If so then the line length might be configured to be too long? 220 chars usually doesn't fit on screens ;)

Reply:
There was someone who wanted it so long, but it doesn't fit on my screen. (4k, only vsc with one side bar).
So I'm fine to break the line earlier.
Reading and editing the editiblevars isn't fun like that.

Reply:
Well, just "someone" shouldn't define the line lengths for the whole project ;) A more modern number might be 100 or 120, more traditional ones are 88.

Reply:
hmm 120 does destroy the layout quit a lot, but formats all editibleVars as we want.
200 is still fine for my screen, editibleVars are still fine, but produce still some mess on other parts